Source/WebKit/UIProcess/WebLockRegistryProxy.cpp

```diff
@@ -47,17 +47,21 @@ WTF_MAKE_TZONE_ALLOCATED_IMPL(WebLockRegistryProxy);
 WebLockRegistryProxy::WebLockRegistryProxy(WebProcessProxy& process)
```

Comment:
At the top of this file, there are two `#define` statements (the message checks) that use `m_process`. Do we need to null check `m_process` inside of these message checks?

Reply:
Non-issue if you apply my other review comment.